blocks/library/freeform/old-editor.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
+import { __ } from 'i18n';
+
+export default class OldEditor extends Component {
+	constructor( props ) {
+		super( props );
+		this.initialize = this.initialize.bind( this );
+		this.onSetup = this.onSetup.bind( this );
+	}
+
+	componentDidMount() {
+		const { baseURL, suffix } = window.wpEditorL10n.tinymce;
+
+		window.tinymce.EditorManager.overrideDefaults( {
+			base_url: baseURL,
+			suffix,
+		} );
+
+		if ( document.readyState === 'complete' ) {
+			this.initialize();
+		} else {
+			window.addEventListener( 'DOMContentLoaded', this.initialize );
+		}
+	}
+
+	componentWillUnmount() {
+		window.addEventListener( 'DOMContentLoaded', this.initialize );
+		wp.oldEditor.remove( this.props.id );
+	}
+
+	componentDidUpdate( prevProps ) {
+		const { id, attributes: { content } } = this.props;
+
+		if ( prevProps.attributes.content !== content ) {
+			window.tinymce.get( id ).setContent( content || '' );
+		}
+	}
+
+	initialize() {
+		const { id } = this.props;
+		const { settings } = window.wpEditorL10n.tinymce;
+
+		wp.oldEditor.initialize( id, {
+			tinymce: {
+				...settings,
+				inline: true,
+				content_css: false,
+				fixed_toolbar_container: '#' + id + '-toolbar',
+				setup: this.onSetup,
+			},
+		} );
+	}
+
+	onSetup( editor ) {
+		const { attributes: { content }, setAttributes } = this.props;
+		const { ref } = this;
+		const initialContent = window.switchEditors.wpautop( content || '' );
+
+		editor.on( 'loadContent', () => editor.setContent( initialContent ) );
+
+		editor.on( 'blur', () => {
+			setAttributes( {
+				content: editor.getContent(),
+			} );
+		} );
+
+		editor.addButton( 'kitchensink', {
+			tooltip: __( 'More' ),
+			icon: 'dashicon dashicons-editor-kitchensink',
+			onClick: function() {
+				const button = this;
+				const active = ! button.active();
+
+				button.active( active );
+				editor.dom.toggleClass( ref, 'has-advanced-toolbar', active );
+			},
+		} );
+
+		/* eslint-disable */
```

Comment:
Could we be more specific here with what we're disabling (i.e. list rule names) to avoid introducing errors, and/or a comment to explain why we're disabling.